### ISSUE TITLE
Cpuminer v4.0.23 stops

### ISSUE THREAD

**Issue by RainbowMiner** [rplant8/cpuminer-opt-rplant#1]
The new windows version of cpuminer v4.0.23 stops shortly after starting stratum. No matter if running cpuminer-ryzen on AMD or cpuminer-aes on Intel or any of the versions.

```
         **********  cpuminer-opt 4.0.23W-aes by rplant ***********
CPU: Intel(R) Pentium(R) CPU G4560 @ 3.50GHz.
SW built on Nov  9 2019 with GCC 8.3.0.
CPU features: SSE2 AES SSE4.2.
SW features: SSE2 AES SSE4.2.
Algo features: SSE2 SHA.
Start mining with SSE2.

[2019-11-09 18:46:56] 4 CPU cores available, 2 miner threads selected.
[2019-11-09 18:46:56] Binding process to cpu mask 5
[2019-11-09 18:46:56] 2 miner threads started, using 'power2b' algorithm.
[2019-11-09 18:46:56] Starting Stratum on stratum+tcp://stratum-eu.rplant.xyz:7022
```

.. whereas the old version v4.0.22 works perfectly.

**Comment by rplant8** [comment]
fixed windows vers 4.0.23a

**Comment by RainbowMiner** [comment]
Thank you!
